test/graphql/mutations/hosts/create_mutation_test.rb

```diff
@@ -322,6 +324,8 @@ class CreateMutationTest < GraphQLQueryTestCase
         let(:context_user) do
           setup_user('show', 'hosts') do |user|
             user.roles << Role.find_by(name: 'Viewer')
+            user.locations << tax_location
+            user.organizations << organization
```

Comment:
Just for future reference: this CP contains these changes since they are original ones. Other related PRs are rather modified due to conflict changes.